i18n/dummy.py

```diff
@@ -138,18 +138,24 @@ class Dummy(BaseDummyConverter):
         """.split()
     )
 
-    # To simulate more verbose languages (like German), pad the length of a string
-    # by a multiple of PAD_FACTOR
-    PAD_FACTOR = 1.33
-
     def pad(self, string):
-        """add some lorem ipsum text to the end of string"""
+        """
+        Add some lorem ipsum text to the end of string to simulate more verbose languages (like German).
+        Padding factor determined by http://www.w3.org/International/articles/article-text-size.en
+        """
         size = len(string)
-        if size < 7:
+        if size <= 10:
             target = size * 3
+        elif size <= 20:
+            target = size * 2
```

Comment:
Yeah - I was just following the W3C's recommendations. I think the idea is that really short strings can end up having much longer than expected translations (eg words that are VERY short in English tend to be abbreviations that languages don't have a shorthand for), but as strings get longer in English they don't have the problem as much.

Did you check out the links in the description? Did I misunderstand the recommendation and it should be done "like taxes" as you say?